Gemfile

```diff
@@ -6,6 +6,7 @@ ruby '3.2.2'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem 'rails', '~> 7.0.6'
 
+gem 'devise'
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if your **sign_in** page comes when your application starts to make your application up to industry standards 👍